tests/Testcontainers.DynamoDB.Tests/Testcontainers.DynamoDB.Tests.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <IsPublishable>false</IsPublishable>
+    <RootNamespace>Testcontainers.DynamoDb.Tests</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
+    <PackageReference Include="coverlet.msbuild" Version="3.2.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5" />
+    <PackageReference Include="xunit" Version="2.4.2" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Testcontainers.DynamoDB\Testcontainers.DynamoDB.csproj" />
+    <ProjectReference Include="$(SolutionDir)tests/Testcontainers.Commons/Testcontainers.Commons.csproj" />
+  </ItemGroup>
+</Project>
```

```suggestion
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFrameworks>net6.0</TargetFrameworks>
    <IsPackable>false</IsPackable>
    <IsPublishable>false</IsPublishable>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1"/>
    <PackageReference Include="coverlet.msbuild" Version="3.2.0"/>
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5"/>
    <PackageReference Include="xunit" Version="2.4.2"/>
  </ItemGroup>
  <ItemGroup>
    <ProjectReference Include="$(SolutionDir)src/Testcontainers.DynamoDB/Testcontainers.DynamoDB.csproj"/>
    <ProjectReference Include="$(SolutionDir)tests/Testcontainers.Commons/Testcontainers.Commons.csproj"/>
  </ItemGroup>
</Project>
```

You need to be careful with the `ProjectReference` path separator. `\` breaks on macOS or Linux.

---

fixed